lib/reporters/json.js

```diff
@@ -78,7 +78,16 @@ function JSONReporter(runner, options = {}) {
 
     runner.testResults = obj;
 
-    var json = JSON.stringify(obj, null, 2);
+    var optionsIndentSize;
+    if (options.reporterOption && 'indentSize' in options.reporterOption) {
+      optionsIndentSize = options.reporterOption.indentSize;
+    } else {
+      optionsIndentSize = 2; // Default Indentation size
+    }
```

Comment:
🧹✨ 

```suggestion
    var optionsIndentSize= options.reporterOption?.indentSize ?? 2;
```

We support Node >=14, which supports [`??` nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#browser_compatibility) and [`?.` optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility).

Reply:
I like it!